### PR TITLE
Libraries: add beatt87 jose-swift library JWT with all JWS algorithms and JWE key algorithms and encryption algorithms, nested JWTs

### PR DESCRIPTION
**Pull request by beatt83**
Add a jose library that adds support of Jose (JWK, JWS, JWE, JWT). https://github.com/beatt83/jose-swift

This library supports fully JWS and JWE, the JWT implementation is built on top of that foundation.

Supported JWT features:

- Signing with all JWS algorithms including EDDSA and ES256K.
- Encrypting with all JWE algorithms including ECDH-1PU.
- Nested JWTs (Encrypted+Encrypted, Encrypted+Signed, Signed+Encrypted).
- Validation of iss, sub, aud, exp, nbf, iat.
- Support of jti, typ and cty.

### CODE REVIEW

**Comment by beatt83** [comment]
Hi @byron-okta. Hope everything is ok.

Ive seen that as well 2 other PRs to add libraries to `jwt.io` website are waiting for more than 6 months already.
(https://github.com/jsonwebtoken/jsonwebtoken.github.io/pull/668)
(https://github.com/jsonwebtoken/jsonwebtoken.github.io/pull/664)

You seem to be the one maintaining this repo, so I would kindly ask if it's possible to review this PR and the other 2 open, or point to whom should we ask to review? 
All this PR's followed the instructions on how request to add libraries, and although I understand the it might take time, 6 months waiting without a reply is a lot.

`jwt.io` is an important resource to share and find JWT libraries for your platform/language.

**Review by localzet — APPROVED** [review]

File: views/website/libraries/19-Swift.json [hunk 1]
```diff
@@ -160,6 +160,41 @@
...
+        "ps384": true,
+        "ps512": true,
+        "eddsa": true
+      },
+      "authorUrl": "https://github.com/beatt83",
+      "authorName": "Gonçalo Frade",
+      "gitHubRepoPath": "beatt83/jose-swift",
+      "repoUrl": "https://github.com/beatt83/jose-swift",
+      "installCommandHtml":
+        ".package(url: \"https://github.com/your-username/jose-swift.git\", .upToNextMinor(from: \"1.1.0\"))"
```
Comment (byron-okta) [inline views/website/libraries/19-Swift.json:197]: Could you please verify if the URL [https://github.com/your-username/jose-swift](https://github.com/your-username/jose-swift) on this command for installing the library is correct? I'm getting a 404 for it.

**Review by byron-okta — CHANGES_REQUESTED** [review]
Hey @beatt83, thank you for contributing to this project. I apologize for taking us so long to review your pull request. 

So far, it is looking good; I left a comment before approving it. Could you please take a look @beatt83 ?

File: views/website/libraries/19-Swift.json (hunk 1 above)
Comment (beatt83) [reply views/website/libraries/19-Swift.json]: Hi @byron-okta . Thx for the review :). Yes it was wrong it's now fixed I took the time to update the version as well for the most recent.

**Review by byron-okta — APPROVED** [review]
LGTM! 
Thank you for your contribution @beatt83

**Comment by beatt83** [comment]
Thank you @byron-okta happy to see this going through :)

**Comment by byron-okta** [comment]
Hey @beatt83 , there is a new policy on the repository that requires all commits to be signed. Could you please sign the commit on this branch 🙏? Here is a document from GH about [singing commits](https://docs.github.com/authentication/managing-commit-signature-verification/about-commit-signature-verification) that can guide you

**Pull request merged**